data/hai ships.txt

```diff
@@ -45,7 +45,7 @@ ship "Aphid"
 	explode "tiny explosion" 10
 	explode "small explosion" 20
 	explode "medium explosion" 15
-	description "The Aphid is a light freighter designed by the Hai. It has only minimal armament."
+	description "The Aphid is the lightest freighter used by the Hai and was for many millennia the backbone of the Hai inter-planetary economy. Given that a captain does not have to fear being attacked in much of Hai space, Aphids are only lightly armed and mainly rely on larger Hai ships for protection."
```

Comment:
~~Given that a captain does not have to deal being attacked in much of Hai space~~

Replace with something similar and less 4th-wall. Not quite sure what though, e.g `The low incidence of piracy...`